logger.go

```diff
@@ -7,22 +7,44 @@ import (
 
 const (
 	TIME_FORMAT = "2006-01-02T15:04:05.999Z"
-	LOG_NAME = "[venom]"
+	LOG_NAME    = "[venom]"
 )
 
-// Logging is the interface which any logging mechanism must satisfy
-// in order to be used as the logging mechanism for a LoggableConfigStore.
-type Logging interface {
+// Logger is the interface which any logging mechanism must satisfy in order to
+// be used as the Logger mechanism wrapped by LogWrapper.
+type Logger interface {
 	Print(...interface{})
 }
 
-// formatLogLine produces a default log line structure for the Logging interface
-// print function to use.
-func formatLogLine(a ...interface{}) string {
-	logLine := fmt.Sprintf("%s%s:", time.Now().UTC().Format(TIME_FORMAT), LOG_NAME)
-	for _, v := range a {
-		logLine += fmt.Sprintf(" %v", v)
+// LogWrapper is the wrapping interface by which a user must adhere in order to
+// be used by the LogableConfigStore.
+type LogWrapper interface {
```

Comment:
I guess I'm not entirely clear why we need a `Logger` and a `LogWrapper`? it seems like we could just make this `LogWrapper` the `Logger` interface, no?

Reply:
I suppose that's true